### ISSUE TITLE
Convert to OSGi bundle

### ISSUE THREAD

**Issue by briantopping** [alexandru/shade#8]
OSGi bundles generally do not depend on OSGi, it just adds key-value pairs to META-INF/MANIFEST.MF. These are generated automatically by build plugins and generally require no maintenance unless there are optional dependencies that the bnd tool cannot discern.

When loading under Apache Karaf, the following feature elements take care of everything:

```
 <bundle>mvn:com.bionicspirit/shade_2.11/1.7.0-SNAPSHOT</bundle>
 <bundle>wrap:mvn:net.spy/spymemcached/2.11.1$Import-Package=org.springframework.beans.factory;com.codahale.metrics;resolution:=optional,shade.memcached.internals,*</bundle>
 <bundle>wrap:mvn:org.monifu/monifu-core_2.11/0.7.0</bundle>
```

**Comment by alexandru** [comment]
@briantopping I've never worked with OSGi bundles, I don't know what that does. 

1. Can this be automated by an SBT plugin?
2. Is anybody still interested? Is this still needed?

Thanks,

**Comment by briantopping** [comment]
I could submit a PR if anyone else wanted it.

**Comment by alexandru** [comment]
@briantopping yes, if it is useful and if it doesn't break anything, then definitely.
